lib/plugins/aws/package/compile/events/cloudFront.js

```diff
@@ -163,6 +163,7 @@ class AwsCompileCloudFrontEvents {
         ViewerProtocolPolicy: {
           enum: ['allow-all', 'redirect-to-https', 'https-only'],
         },
+        TrustedKeyGroups:{type: 'array'}
```

Comment:
Let's specify what type of items we expect in that array.

Reply:
I guess what @pgrzesik proposing in a change like this:

```suggestion
        TrustedKeyGroups:{type: 'array', items: {type: 'string'}}
```

So @Patil2099 feel free to add this change. If not... I'd be happy to finish it